wallet-webapps-admin/src/main/webapp/vue-app/components/admin/WalletAdminApp.vue

```diff
@@ -22,36 +22,10 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
     flat>
     <main>
       <v-layout column>
-        <v-flex>
-          <v-card class="applicationToolbar mb-3" flat>
-            <v-flex class="ps-3 pe-3 pt-2 pb-2">
-              <strong>{{ $t('exoplatform.wallet.title.walletAdministration') }}</strong>
-            </v-flex>
-          </v-card>
-        </v-flex>
-        <v-flex class="white text-center" flat>
-          <div v-if="error && !loading" class="alert alert-error v-content">
-            <i class="uiIconError"></i>{{ error }}
-          </div>
-
-          <v-dialog
-            v-model="loading"
-            persistent
-            hide-overlay
-            width="300">
-            <v-card color="primary" dark>
-              <v-card-text>
-                {{ $t('exoplatform.wallet.label.loading') }} ...
-                <v-progress-linear
-                  indeterminate
-                  color="white"
-                  class="mb-0" />
-              </v-card-text>
-            </v-card>
-          </v-dialog>
```

Comment:
We need to keep the loading and the error boxes.

Reply:
done!